### PR TITLE
ui: add usage log table (PROJQUAY-6702)

### PR DESCRIPTION
**Pull request by Marcusk19**
Adds a table for logs in the repository and organization settings. 

Screens:
Screenshot 2024-02-21 at 4 14 44 PM

https://github.com/quay/quay/assets/47163063/2b295614-a8b0-423e-a0c0-30fb653b8574

### CODE REVIEW

File: web/tsconfig.json [hunk 1]
```diff
@@ -11,9 +11,8 @@
     "outDir": "dist",
     "allowSyntheticDefaultImports": true,
     "sourceMap": true,
+    "types": ["cypress", "node"]
   },
-  "include": [
-    "src"
-  ],
-  "exclude": [ "node_modules" ]
+  "include": ["src", "**/*.ts"],
```
Comment (dmage) [inline web/tsconfig.json:16]: shouldn't we have all *.ts files in src/?

Comment (Marcusk19) [reply web/tsconfig.json:16]: yeah, but for some reason my linter was showing me errors for *.cy.ts files without this include

Comment (dmage) [reply web/tsconfig.json:16]: ah, ok, cypress tests are in cypress/e2e/, not src/

**Review by dmage — APPROVED** [review]

**Review by arborite-rh — APPROVED** [review]
lgtm

**Pull request merged**